pipenv/core.py

```diff
@@ -2409,9 +2409,14 @@ def do_check(three=None, python=False, system=False, unused=False, args=None):
         python = which('python')
     else:
         python = system_which('python')
+
+    try:
+        ignore = ' '.join(args)
```

Comment:
Using `ignore` here is confusing, because there could be other args. Consider using `safety_args` instead?

Reply:
@techalchemy?